pkgs/development/tools/analysis/coz/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, fetchFromGitHub
+, libelfin
+, ncurses
+, nodejs
+, python3
+, python3Packages
+}:
+stdenv.mkDerivation rec {
+  pname = "coz";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "plasma-umass";
+    repo = "coz";
+    rev = version;
+    sha256 = "0a55q3s8ih1r9x6fp7wkg3n5h1yd9pcwg74k33d1r94y3j3m0znr";
+  };
+
+  buildInputs = [
+    nodejs
```

Comment:
`node` is only required for the viewer, which is not built from this expression.

Reply:
fixed in 5a1aef8468685e3586be5b8e53cbdb91ade7fe64. I'll do the frontend later